docs/guides/migrations.md

```diff
@@ -9,6 +9,11 @@ kb_sync_latest_only
 
 ## 2.4 to 3.0
 
+With the 2.4.1 Hotfix we introduced a more fixed Node.js version handling.
+For the docker containers and as the projects engine we set `Node.js 16.16.0` and `npm 8.11.0` to more specifically refer to the versions used and tested.
```

Comment:
Sorry I have trouble understanding that sentence. May I suggest:
`We set Node.js 16.16.0 and npm 8.11.0 as our application runtime and package management versions.`